web/src/app/schedules/ScheduleRuleList.js

```diff
@@ -111,11 +111,16 @@ export default class ScheduleRuleList extends React.PureComponent {
     return (
       <React.Fragment>
         <PageActions>
-          <FilterContainer onReset={() => this.props.resetFilter()}>
-            <Grid item xs={12}>
-              <ScheduleTZFilter scheduleID={this.props.scheduleID} />
-            </Grid>
-          </FilterContainer>
+          <ScheduleTZFilter
+            scheduleID={this.props.scheduleID}
+            render={(children) => (
+              <FilterContainer onReset={() => this.props.resetFilter()}>
```

Comment:
What's going on here makes sense, but having the Wrapper/Container element nested in the Filter as a prop reads a little off. What is the main benefit in reversing the children logic here into a render prop?